test/package.json

```diff
@@ -13,6 +13,7 @@
     "esbuild": "^0.17.16",
     "express": "^4.18.2",
     "iconv-lite": "^0.6.3",
+    "jest": "^29.5.0",
```

Comment:
is this intentional?

Reply:
accidentally added when i was trying to run the node event emitter test with jest. good catch.